test/env/EnvTest.gmk

```diff
@@ -27,8 +27,7 @@ ENV.files := $(shell find $(TESTDIR)/env -type f)
 
 $(BUILDDIR)/EnvTest.simple.ok: $(ENV.files) \
 	    $(JTREG_IMAGEDIR)/lib/javatest.jar \
-	    $(JTREG_IMAGEDIR)/lib/jtreg.jar \
-	    $(JTREG_IMAGE_JAVAHELP_JAR) \
+	    $(JTREG_IMAGEDIR)/lib/jtreg.jar
```

Comment:
Yes, will fix.   I guess it executed because the dependency on the next like was "just" the executable sc riot, so the dependency became a no-args invocated of the script, i.e. as the first rule.